docs/transformer.md

```diff
@@ -71,6 +71,12 @@ The end marker is:
 <!-- INTERACTIVE exec END -->
 ```
 
+> #### NOTE
+>
+> By default, the tool defaults to make code fenced with `bash` as executable. Meaning you do not need to specify the `<!-- INTERACTIVE exec START/STOP -->` directives for bash code blocks. It is possible to override this behaviour by adding the directive such as `<!-- INTERACTIVE copy START/STOP -->` to the fenced code block as directives always take precedence.
```

Comment:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts


```suggestion
> [!NOTE]
> By default, the tool makes `bash` fenced code blocks executable so you don't need `<!-- INTERACTIVE exec START/STOP -->` directives for bash code blocks.
> You can override this behavior with the `<!-- INTERACTIVE copy START/STOP -->` directives which take precedence over the default behavior.
```